internal/analysis/pairwise_test.go

```diff
@@ -802,3 +801,71 @@ func TestPairwiseAnalysisFailureBase(test *testing.T) {
 			expectedMessageSubstring, results[key].FailureMessage)
 	}
 }
+
+func TestGetJplagOptions(test *testing.T) {
```

Comment:
This test seems like it should be in the jplag package.

(And if this is true, then we don't have to export the parsing function.)